cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,48 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+
+const MAX_PAGE_WEIGHT = 100;
+
+export default ({ path, pageType }) => {
+  describe('Page weight', () => {
+    const allRequests = [];
+    const getPageSizeInBytes = request => {
+      const getSize = `echo ${JSON.stringify(request)} | gzip | wc -c`;
+      if (typeof request === 'string') {
```

Comment:
What other types of requests are there? I wonder if we get more results if we remove this?

Reply:
I can remove this and its fine as those that arent a string resolve to NaN when trying to count their bytes, i put it in initally because some of the get requests are just firing off events to ati which we're not concerned with, and those are objects.

Reply:
I think they dont count as they're after the inital page load - i could filter them out, any that are just pinging events have a body thats an ArrayBuffer so we could filter on that instead of a string, but seems like it's just doing the same thing as above? I think any stuff we care about would have a string body

Reply:
Shall we leave it for the moment, and get a second opinion? Because the ATI page view might count towards page size (in which case we might want to convert the Array Buffer to a string and get the size), or we filter it (and other requests like the chartbeat ping) out.

Reply:
Yeahhh good idea 😊